### DISCUSSION TITLE
正版购买咋不行了

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3489] [讨论]
我用微软商店，结果不停报错让我稍后再试，Steam++也用了……
图片传不上（）

**Comment by allMagicNB** [comment]
去Xbox买

**Reply by SqrtTan** [reply]
试过了，但是一样，甚至连界面都加载不全……

**Reply by zkitefly** [reply]
https://www.xbox.com/zh-HK/games/store/minecraft/9MVXMVT8ZKWC

**Reply by allMagicNB** [reply]
https://www.xbox.com/zh-CN/games/store/minecraft-java-bedrock-edition-for-pc/9NXP44L49SHJ/0010/9NSPXMS6M2MF

**Reply by SqrtTan** [reply]
界面这样：
https://pan.baidu.com/s/18SkC5u5_yfS4yvfjJKcf3Q?pwd=1234
图片传不上来是为啥（

**Reply by zkitefly** [reply]
> 界面这样：
> https://pan.baidu.com/s/18SkC5u5_yfS4yvfjJKcf3Q?pwd=1234
> 图片传不上来是为啥（

大概率就是你电脑访问外国网络不通畅

**Reply by zkitefly** [reply]
https://www.microsoft.com/store/productId/9NXP44L49SHJ

**Reply by SqrtTan** [reply]
我认为我需要七根木棍……问题是找不到（）

**Reply by SqrtTan** [reply]
~啊？还真敢发？~
要么删了吧（

**Reply by allMagicNB** [reply]
> ~啊？还真敢发？~
> 要么删了吧（

你用完了我就删
